### DISCUSSION TITLE
Predominant isoform mean(value)?

### DISCUSSION THREAD

**Discussion by kubu4** [sr320/ceabigr#71] [Q&A] [answered]
@sr320 

Was going through the R code for determining predominant isoforms and wasn't sure what the purpose of this line is:

https://github.com/sr320/ceabigr/blob/51ce81ca7d6aa338972789ecb47d4a5af185fd18/code/42-predominant-isoform.Rmd#L116

Basically, I'm not entirely sure what that chunk of code is doing. Are you able to explain it a bit?

**Comment by kubu4** [comment, marked as answer]
Wouldn't we want that to be `max(value)`? I see that there's a `which.max()` call below that, but if it's calculating a mean, then there is no maximum value any more, right?

**Reply by sr320** [reply]
What I did was first find mean value across samples (as there are several individual values. Then once mean value of isoform (across samples) was determined I set out to ID the max mean value.

**Reply by kubu4** [reply]
Ah, right! Thanks!
